packages/salesforcedx-vscode-soql/test/vscode-integration/queryDataView/queryDataFileService.test.ts

```diff
@@ -58,7 +58,7 @@ describe('Query Data File Service', () => {
   afterEach(() => {
     // delete the query-results directory and its files.
     // @ts-ignore
-    fs.rmdirSync(testResultsDirPath, { recursive: true });
+    fs.rm(testResultsDirPath, { recursive: true });
```

Comment:
`(node:488) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead`